ChangeLog.md

```diff
@@ -140,6 +140,8 @@
   * Change Docker postgresql to do initializations/migrations at build (Dave Mussulman).
 
   * Change the example course to be available to any institution (Matt West).
+  
+  * Change the HTML element documentation to list elements under two categories with example links (James Balamuta).
```

Comment:
Many apologies for letting this PR sit around so long. It needs an update to kebab-case element names and the forthcoming kebab-case change for attribute names (which I think @nwalters512 is planning to push within the next day or so.

Would you mind rebasing this to current master and liaising with @nwalters512 about kebab-case attributes? I promise to merge it fast after this :-)

Reply:
I'll take care of it later this weekend.

I'm also planning to switch the names of the examples from `elementExamples` to `examplesElement` so they appear together in `questions`.

Reply:
Awesome! And good idea.

Reply:
Maybe you can also check on @mfsilva22's new elements to standardize on naming for these example questions?